df.announcements.xml

```diff
@@ -1241,7 +1241,7 @@
         <int32_t name='activity_event_id' init-value='-1' comment='same as field in report'/>
         <int32_t name='speaker_id' ref-target='unit'/>
         <bitfield base-type='uint8_t' name='flags'>
-            <flag-bit name='hostile_combat' comment='hunting or combat report, not sparring'/>
+            <flag-bit name='is_sparring' comment='determined by unit_action_data_attack.flags.is_sparring'/>
```

Comment:
bay12 name for this flag bit is `SPARRING_EVENT`:
`#define ANNOUNCEMENT_INFO_FLAG_SPARRING_EVENT BIT1`